internal/app/api/handler/actvisitassoc/act_visit_assoc.go

```diff
@@ -123,7 +120,8 @@ func (h *Handler) Delete(w http.ResponseWriter, r *http.Request) {
 
 // DeleteByVisitID handle delete activity associate HTTP Request
 func (h *Handler) DeleteByVisitID(w http.ResponseWriter, r *http.Request) {
-	if err := h.srv.DeleteByVisitID(r.Context(), mux.Vars(r)["visitID"]); err != nil {
+	queryValues := r.URL.Query()
```

Comment:
Should get query argument and validate first before calling service.
Same for other places.